### ISSUE TITLE
Missing Label

### ISSUE THREAD

**Issue by mgifford** [formbricks/formbricks#1552]
### Issue Summary

Title: WCAG 4.1.2: Ensures every form element has a label (.ml-2)
Tags: Accessibility, WCAG 4.1.2, label

Issue: Ensures every form element has a label (label - https://accessibilityinsights.io/info-examples/web/label)

Target application: Formbricks - https://app.formbricks.com/onboarding

Element path: .ml-2

Snippet: <input class="ml-2 mr-2 h-10 w-32 flex-1 border-0 bg-transparent text-slate-500 outline-none focus:border-none" spellcheck="false" value="64748b">

How to fix: 
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element has no placeholder attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 118.0.2088.76

====

This accessibility issue was found using Accessibility Insights for Web 2.40.0 (axe-core 4.7.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### Steps to Reproduce

Still trying to go through https://app.formbricks.com/onboarding

### Expected behavior

All forms should have labels

### Screenshots

![Screenshot 2023-11-01 at 7 17 02 PM](https://github.com/formbricks/formbricks/assets/116832/ee1874c6-0f01-45d1-b2bb-6a6f634b618b)

### Environment

- [ ] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```
